### PR TITLE
feat: IDP-1316_alert_OPEX_workbook

### PR DESCRIPTION
**Pull request by sbellezz**
### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

#### Has This Been Tested?

- [ ] Yes
- [ ] No

### Other information
Target: 
```
azurerm_monitor_metric_alert.ErrorsTopic
azurerm_monitor_scheduled_query_rules_alert_v2.Availability
azapi_resource.idpay_workbook
```

#### How Has This Been Tested?

- [ ] Unit Test Suite
- [ ] Integration Test Suite
- [ ] Api Tests
- [ ] Load Tests

[Link to test results](https://pagopa.atlassian.net/browse/RTD-XXXX)

---

### CODE REVIEW

File: src/domains/idpay-app/12_alert.tf [hunk 1]
```diff
@@ -264,3 +264,89 @@ data
 
   tags = var.tags
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert_v2" "AvailabiltyIdPay" {
```
Comment (and-mora) [inline src/domains/idpay-app/12_alert.tf:268]: may you fix the typo in `Availability` word?

File: src/domains/idpay-app/12_alert.tf [hunk 2]
```diff
@@ -264,3 +264,89 @@ data
...
+
+  count               = var.idpay_alert_enabled ? 1 : 0
+  name                = "${local.project}-AvailabiltyIdPay"
```
Comment (and-mora) [inline src/domains/idpay-app/12_alert.tf:271]: may you fix the typo in `Availability` word?

File: src/domains/idpay-app/12_alert.tf [hunk 3]
```diff
@@ -264,3 +264,89 @@ data
...
+
+    failing_periods {
+      minimum_failing_periods_to_trigger_alert = 1
+      number_of_evaluation_periods             = 1
+    }
+  }
+
+  auto_mitigation_enabled          = false
+  workspace_alerts_storage_enabled = false
+  description                      = "Trigger alert when Idpay's Availabilty is less than 0.99"
```
Comment (and-mora) [inline src/domains/idpay-app/12_alert.tf:306]: may you fix the typo in `Availability` word?

File: src/domains/idpay-app/12_alert.tf [hunk 4]
```diff
@@ -264,3 +264,89 @@ data
...
+  display_name                     = "${var.domain}-${var.env_short}-AvailabiltyIdPay"
```
Comment (and-mora) [inline src/domains/idpay-app/12_alert.tf:307]: may you fix the typo in `Availability` word?

**Review by and-mora — CHANGES_REQUESTED** [review]

**Comment by and-mora** [comment]
couple of questions:
- the alarms have been tested?
- has been applied in dev?

**Comment by sbellezz** [comment]
@and-mora yes, they've been applied in dev. The Availability alert cannot be tested in dev because the metric isn't available in this environment, the other one has been tested.

**Comment by and-mora** [comment]
@sbellezz targets in description do not work, may you update it?
 
the alerts has been applied in uat, I'd like a feedback on testing if you can provide it. 
the workbook is not yet applied but working on it.

**Comment by and-mora** [comment]
@sbellezz it is needed another run on `pre-commit` since the static analysis is failing, thank you

**Review by and-mora — DISMISSED** [review]

**Review by and-mora — APPROVED** [review]

**Review by TommasoLencioni — APPROVED** [review]

**Pull request merged**